scalability_and_performance/index.adoc

```diff
@@ -0,0 +1,63 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="index"]
+= {product-title} Scalability and Performance overview
+include::_attributes/common-attributes.adoc[]
+:context: index
+
+// ifndef::openshift-dedicated,openshift-rosa[]
+{product-title} offers best practices and utilities to help you optimize the performance and scale of your {product-title} clusters. This book in the documentation provides information on recommended performance and scalability practices, reference design specifications, optimization and low latency tuning.
```

Comment:
"This book" is self referential language, avoid using when possible. 

"The following documentation provides information on...."

Reply:
done, thank you!